catalog/errors.go

```diff
@@ -31,4 +31,5 @@ var (
 	ErrNonDirectNotSupported         = errors.New("non direct diff not supported")
 	ErrNonDirectMergeNotSupported    = errors.New("merge suported only between branches that are parent-child or child-parent")
 	ErrSameBranchMergeNotSupported   = errors.New("same branch merge not supported")
+	ErrMergeWorkerRollback           = errors.New("merge worker exit because of diff error")
```

Comment:
can remove this one?

Reply:
removed it and two others that are not needed